src/Sulu/Bundle/AdminBundle/Resources/js/components/CircleSelection/CircleSelection.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import type {Node} from 'react';
+import React from 'react';
+import withContainerSize from '../withContainerSize';
+import CircleSelectionRenderer from './CircleSelectionRenderer';
+import type {SelectionData} from './types';
+import circleSelectionStyles from './circleSelection.scss';
+
+type Props = {
+    children?: Node,
+    disabled: boolean,
+    filled: boolean,
```

Comment:
`skin="circle"` and `skin="point"`?

Reply:
Sounds good to me. Are there also other props that are mutually exclusive to circles and points? Maybe we could get rid of some unexpected states, if we also base them on such a property. Maybe it should even be a mode then? WDYT?

Reply:
There's only the `filled` prop and a `resizable` prop ... But I would not combine those, because then you can't create "filled" circles anymore, which are also resizable. But what about `skin="filled"` and `skin="outlined"` or something like that in combination with the `resizable` prop?

Reply:
Okay, then `skin` is fine to me. Would say allowed values are `circle` and `point`, as `filled` and `outlined` do again only describe a single characteristic. The `resizable` prop can also continue to exist, if we also need any of those skins in a resizable and not resizable variant :+1:

Reply:
IMO `skin="point"` would indicate, that it's not resizable, so I would still stick with `filled` and `outlined`, because that's the only thing the prop changes

Reply:
Okay, can live with that. Would still slightly prefer the other way, because you are also changing the `color` styling based on that.